lib/passageidentity/auth.rb

```diff
@@ -76,15 +76,13 @@ def validate_jwt(token)
         )
       end
 
-      audiences = [@auth_origin, @app_id]
-
       claims =
         JWT.decode(
           token,
           nil,
           true,
           {
-            aud: audiences,
+            aud: [@app_id],
```

Comment:
From the spec test [here](https://github.com/jwt/ruby-jwt/blob/b006395d85f6f9a86b9df21a4dbcbda023e47431/spec/jwt/verify_spec.rb#L30) , it looks like the array wrapper can be flattened out:
```suggestion
            aud: @app_id,
```

Reply:
updated in [149e9a1](https://github.com/passageidentity/passage-ruby/pull/147/commits/149e9a1512329e863bbc4c91c494a0bd8d6d1f58)